topics/admin/tutorials/tiaas/tutorial.md

```diff
@@ -135,6 +135,11 @@ This tutorial will go cover how to set up such a service on your own Galaxy serv
 >    +    privs: SELECT,INSERT
 >    +  - database: galaxy
 >    +    roles: tiaas
+>    +    objs: group_role_association
+>    +    type: table
+>    +    privs: DELETE
+>    +  - database: galaxy
+>    +    roles: tiaas
```

Comment:
could you call out the new feature in a tip box? I think it's important for admins to know that'll be happening in the background

Reply:
Good idea, done!